code-of-conduct.md

```diff
@@ -55,6 +55,13 @@ Harassment includes, but is not limited to:
 * Supporting nature-based (or similar, non-social) explanations for professional inequalities, wage gaps, or the experience of inequal treatment (either as an individual or as a group)
 * "Doxxing" - the publication of personal identifying information or photos of a Slack member that is otherwise private
 
+Other inappropriate content
+---------------------------
+
+This Slack should be considered safe for general consumption. As a rule of thumb, before you post anything, consider if it would be appropriate for the widest possible audience: there is no minimum age restriction for joining this Slack and people from all walks of like – as well as faiths and beliefs – are welcome here.
```

Comment:
```suggestion
This Slack should be considered safe for general consumption. As a rule of thumb, before you post anything, consider if it would be appropriate for the widest possible audience: there is no minimum age restriction for joining this Slack and people from all walks of life – as well as faiths and beliefs – are welcome here.
```